internal/tester_definition.go

```diff
@@ -49,6 +50,7 @@ var testerDefinition = tester_definition.TesterDefinition{
 		{
 			Slug:     "nd2",
 			TestFunc: testDownloadPiece,
+			Timeout:  30 * time.Second,
```

Comment:
@glokta1 we haven't needed this before - are you sure this is required? Does CI pass without it?

Reply:
@rohitpaulk Locally, the request to tracker URL kept timing out otherwise. I'll try pushing without the extra timeout and see if it works.